Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs

```diff
@@ -27,6 +27,10 @@
 FileStream outputStream = new FileStream(Path.GetFullPath(@"Output/Output.docx"), FileMode.Create, FileAccess.Write);
 mainDocument.Save(outputStream, FormatType.Docx);
 
+//Dispose the streams.
+fileStream1.Dispose();
+fileStream.Dispose();
+
```

Comment:
Use proper variable names and modify proper input document name.

Reply:
Addressed